pkg/flowkit/go.mod

```diff
@@ -7,14 +7,14 @@ require (
 	github.com/gosuri/uilive v0.0.4
 	github.com/joho/godotenv v1.4.0
 	github.com/manifoldco/promptui v0.9.0
-	github.com/onflow/cadence v0.24.6
+	github.com/onflow/cadence v0.24.2-0.20220627202951-5a06fec82b4a
 	github.com/onflow/flow-core-contracts/lib/go/templates v0.11.2-0.20220513155751-c4c1f8d59f83
-	github.com/onflow/flow-emulator v0.33.3
-	github.com/onflow/flow-go v0.26.12
-	github.com/onflow/flow-go-sdk v0.26.4
+	github.com/onflow/flow-emulator v0.33.4-0.20220705151023-2cc6a4f25a20
+	github.com/onflow/flow-go v0.25.13-0.20220706165514-abf4535fe946
+	github.com/onflow/flow-go-sdk v0.26.5-0.20220629191626-900f9f91bffc
```

Comment:
Looks like `flow-go` and `cadence` were bumped down. Is that intentional?

Reply:
This is because the "master" branch has not been updated to the current release version and is still on the old one. At least for Cadence master has all the commits of the latest release, not sure about flow-go.

Reply:
Yes it should be.